docs/the-book/contributing/key-contributors.md

```
@@ -0,0 +1,81 @@
+# Key Contributors
+
+### Introduction
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)